.php_cs

```diff
@@ -1,25 +1,59 @@
 <?php
 
-return Symfony\CS\Config\Config::create()
-    ->setUsingLinter(false)
-    ->setUsingCache(true)
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
-    ->fixers([
-        'concat_with_spaces',
-        '-concat_without_spaces',
-        '-empty_return',
-        '-phpdoc_params',
-        '-phpdoc_separation',
-        '-phpdoc_to_comment',
-        '-spaces_cast',
-        '-blankline_after_open_tag',
-        '-single_blank_line_before_namespace',
+if (class_exists('Symfony\CS\Config\Config')) {
+    // PHP-CS-Fixer 1.x syntax (deprecated)
```

Comment:
You can remove 1.x config, it will produce different result, so we can not support that.